.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
     - master
     - "3.0"
+    - "2.18"
     - "2.17"
```

Comment:
Do we need to keep building snapshots for 2.17 🤔?

Reply:
yes - we are going to continue doing 2.17 releases for a while and need snapshots to allow testing before releases are made

Reply:
I think that inclusion (or not) in 2.18 branch has no effect on CI for 2.17 branch (but rather `main.yaml` in `2.17` branch itself)?

And yes, we would prefer having SNAPSHOTs published from 2.17 too.